### PR TITLE
Apply concept template to extend-k8s and configuration concepts

### PR DESCRIPTION
**Pull request by mistyhacks**
Fixes problems with doubled bullet points in TOC

/cc @ajaeger 
/assign @steveperry-53

### CODE REVIEW

**Comment by mistyhacks** [comment]
OK, these look good to me now. I limited it to the Extending Kubernetes and Configuration subdirectories of concepts.

/hold cancel

File: content/en/docs/concepts/configuration/secret.md [hunk 1]
```diff
@@ -1,16 +1,23 @@
 ---
 reviewers:
 - mikedanese
-title: Secrets
+title:
```
Comment (ajaeger) [inline content/en/docs/concepts/configuration/secret.md]: Why did you remove the title? Now https://deploy-preview-8995--kubernetes-io-master-staging.netlify.com/docs/concepts/configuration/secret/ does not show a page title (only "Kubernetes" - not as previously "Kubernetes - Secrets")

Comment (mistyhacks) [reply content/en/docs/concepts/configuration/secret.md]: That was inadvertent .... let me put it back.

Comment (mistyhacks) [reply content/en/docs/concepts/configuration/secret.md]: Fixed.
